### PR TITLE
fix macrel faa input

### PR DESCRIPTION
**Pull request by Darcy220606**
This PR fixes the `MACREL` input issue in the pipeline amp workflow. The issue was that `MACREL` module is currently using fna and running its own annoations therfore when `MACREl` is run alone in the pipeline `AMPCOMBI` was not activated as it was missing one of its input = faa. To fix this i added the `faa` generated by `MACREl` into the summary channel.  This closes issue #241 
`MACREL` alse needs updating in `FUNCSCAN/MODULES` an issue [#3001](https://github.com/nf-core/modules/pull/3001) was also created.

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by jfy133** [comment]
@Darcy220606 please just check locally it works correctly now I've resolved the merge conflict, then you can merge :)

File: subworkflows/local/amp.nf [hunk 1]
```diff
@@ -41,9 +41,13 @@ workflow AMP {
...
-        GUNZIP_MACREL ( MACREL_CONTIGS.out.amp_prediction )
-        ch_versions = ch_versions.mix(GUNZIP_MACREL.out.versions)
-        ch_ampresults_for_ampcombi = ch_ampresults_for_ampcombi.mix(GUNZIP_MACREL.out.gunzip)
+        GUNZIP_MACREL_PRED ( MACREL_CONTIGS.out.amp_prediction )
+        GUNZIP_MACREL_ORFS ( MACREL_CONTIGS.out.all_orfs )
+        ch_versions = ch_versions.mix(GUNZIP_MACREL_PRED.out.versions)
+        ch_versions = ch_versions.mix(GUNZIP_MACREL_ORFS.out.versions)
+        ch_ampresults_for_ampcombi = ch_ampresults_for_ampcombi.mix(GUNZIP_MACREL_PRED.out.gunzip)
+        ch_macrel_faa = ch_macrel_faa.mix(GUNZIP_MACREL_ORFS.out.gunzip)
+        ch_faa_for_ampcombi = ch_faa_for_ampcombi.mix(ch_macrel_faa)
```
Comment (jfy133) [inline subworkflows/local/amp.nf:50]: Hrashil Alignment ™️ plz (all `=` in the same place)

Comment (Darcy220606) [reply subworkflows/local/amp.nf]: Done!

File: subworkflows/local/amp.nf [hunk 2]
```diff
@@ -33,24 +33,28 @@ workflow AMP {
...
-        ch_ampresults_for_ampcombi = ch_ampresults_for_ampcombi.mix(GUNZIP_MACREL.out.gunzip)
+        ch_versions                 = ch_versions.mix(MACREL_CONTIGS.out.versions)
+        GUNZIP_MACREL_PRED ( MACREL_CONTIGS.out.amp_prediction )
+        GUNZIP_MACREL_ORFS ( MACREL_CONTIGS.out.all_orfs )
+        ch_versions                 = ch_versions.mix(GUNZIP_MACREL_PRED.out.versions)
+        ch_versions                 = ch_versions.mix(GUNZIP_MACREL_ORFS.out.versions)
+        ch_ampresults_for_ampcombi  = ch_ampresults_for_ampcombi.mix(GUNZIP_MACREL_PRED.out.gunzip)
+        ch_macrel_faa               = ch_macrel_faa.mix(GUNZIP_MACREL_ORFS.out.gunzip)
+        ch_faa_for_ampcombi         = ch_faa_for_ampcombi.mix(ch_macrel_faa)
     }
```
Comment (louperelo) [inline subworkflows/local/amp.nf:52]: Why did you choose to first create the `ch_macrel_faa`? For readability?
Wouldn't this have had the same effect (two lines less code): 
`ch_faa_for_ampcombi = ch_faa_for_ampcombi.mix(GUNZIP_MACREL_ORFS.out.gunzip)` ?

**Review by louperelo — COMMENTED** [review]
Continues looking good.

**Pull request merged**